NewApp/ngsa-csharp/Ngsa.LodeRunner/baseline.json

```diff
@@ -104,7 +104,7 @@
-    {"path":"/api/movies?year=2026","perfTarget":{"category":"SearchMovies"},"validation":{"statusCode":400, "contentType":"application/problem+json","jsonObject":[{"field":"type","value":"https://github.com/retaildevcrews/ngsa/blob/main/docs/ParameterValidation.md#movies-api"},{"field":"title","value":"Parameter validation error"},{"field":"detail","value":"One or more invalid parameters were specified."},{"field":"status","value":400},{"field":"instance","value":"/api/movies?year=2026"},{"field":"validationErrors","validation":{"jsonArray":{"count":1,"forEach":[{"jsonObject":[{"field":"code","value":"InvalidValue"},{"field":"target","value":"year"},{"field":"message","value":"The parameter 'year' should be between 1874 and 2025."}]}]}}}]}},
+    {"path":"/api/movies?year=2027","perfTarget":{"category":"SearchMovies"},"validation":{"statusCode":400, "contentType":"application/problem+json","jsonObject":[{"field":"type","value":"https://github.com/retaildevcrews/ngsa/blob/main/docs/ParameterValidation.md#movies-api"},{"field":"title","value":"Parameter validation error"},{"field":"detail","value":"One or more invalid parameters were specified."},{"field":"status","value":400},{"field":"instance","value":"/api/movies?year=2027"},{"field":"validationErrors","validation":{"jsonArray":{"count":1,"forEach":[{"jsonObject":[{"field":"code","value":"InvalidValue"},{"field":"target","value":"year"},{"field":"message","value":"The parameter 'year' should be between 1874 and 2025."}]}]}}}]}},
     {"path":"/api/movies?year=2020.1","perfTarget":{"category":"SearchMovies"},"validation":{"statusCode":400, "contentType":"application/problem+json","jsonObject":[{"field":"type","value":"https://github.com/retaildevcrews/ngsa/blob/main/docs/ParameterValidation.md#movies-api"},{"field":"title","value":"Parameter validation error"},{"field":"detail","value":"One or more invalid parameters were specified."},{"field":"status","value":400},{"field":"instance","value":"/api/movies?year=2020.1"},{"field":"validationErrors","validation":{"jsonArray":{"count":1,"forEach":[{"jsonObject":[{"field":"code","value":"InvalidValue"},{"field":"target","value":"year"},{"field":"message","value":"The parameter 'year' should be between 1874 and 2025."}]}]}}}]}},
```

Comment:
Need to update the message too in these tests "The parameter 'year' should be between 1874 and 2025." It should now be 2026.

Reply:
This may be a code change too

Reply:
Good catch - fixed